Aptoide/aptoide-client-v8
# [ASV-1906] Remove IronSource

**What does this PR do?**

   This PR aims at removing IronSource from our codebase. This is an ad dependency that we don't need anymore.

**Database changed?**

 No

**Where should the reviewer start?**

- [ ] MainActivity.java

**How should this be manually tested?**

  Open the app and check that everything is ok with the ads. Also, run this in release to make sure everything is ok with proguard.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1906](https://aptoide.atlassian.net/browse/ASV-1906)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
